bin/concert

```diff
@@ -435,6 +436,7 @@ class StartCommand(Command):
 
             if not module:
                 from concert.quantities import q
+            concert.base._INSTANCE_CACHE = True
```

Comment:
I want to ensure nothing is cached before the devices are fully created and a session is started, because there might be some device initialized inside concert or elsewhere with the user-facing name defined in a session later. E.g. the dummy camera sets its attributes in a constructor, which would result in a name *self* which is not what we want. On the other hand, we need caching because the search is pretty slow, in the order of 10 ms.
The question is if this is the correct place to indicate a session has started.